backend/kernelCI_app/constants/localization.py

```diff
@@ -85,21 +85,21 @@ class DocStrings:
 
     # KCI Summary related descriptions
     REGRESSIONS_GROUP = "Regressions are grouped by hardware, config, and path."
```

Comment:
I guess this can be removed as well
```suggestion
    REGRESSIONS_GROUP = "Regressions are grouped by hardware, config, and path."
```

Reply:
Resolved